scr/Parking-Zone.Data/Models/AppUser.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using Microsoft.AspNetCore.Identity;
+using Parking_Zone.Domain.Entities;
 
 namespace Parking_Zone.Data.Models
 {
     public class AppUser : IdentityUser
     {
         [PersonalData]
         public string Name { get; set; }
+
+        public virtual ICollection<Reservation> Reservations { get; set; }
```

Comment:
why the reservations property in AppUser

Reply:
if you know I separated layers, and i put AppUser into data layer so there was problem to form relationship between Reservation and AppUser because of different layers, and i put reservations as navigation property to build relationship with fluent API